packages/cli/create-strapi-app/src/create-strapi.ts

```diff
@@ -254,6 +255,10 @@ async function createApp(scope: Scope) {
       logger.fatal('Failed to start your Strapi application');
     }
   }
+
+  if (isABTestEnabled) {
+    await trackUsage({ event: 'didEnabledABTest', scope });
```

Comment:
I would change the event name here to `didEnableABTest` just makes more sense grammatically
but overall the code looks good